Conceptpower+Spring/src/main/java/edu/asu/conceptpower/web/ReviewRequestController.java

```diff
@@ -0,0 +1,36 @@
+package edu.asu.conceptpower.web;
+
+import java.security.Principal;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.validation.BindingResult;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+
+import edu.asu.conceptpower.app.core.impl.RequestsManager;
+import edu.asu.conceptpower.core.ReviewStatus;
+import edu.asu.conceptpower.core.RequestJsonResponse;
+import edu.asu.conceptpower.core.ReviewRequest;
+
+@Controller
+public class ReviewRequestController {
+ 
+    @Autowired
+    private RequestsManager requestsMgr;
+           
+    @RequestMapping(value = "/auth/request/add", method = RequestMethod.POST )
+    public @ResponseBody RequestJsonResponse addNewReviewRequest( @ModelAttribute(value="reviewRequest") ReviewRequest reviewRequest, BindingResult result,Principal principal) {
```

Comment:
why are you injecting BindingResult if you dont' use it? You probably want to validate the review request, and then you need the binding results.